index.js

```diff
@@ -33,8 +33,14 @@ function fastifyStatic (fastify, opts, next) {
     maxAge: opts.maxAge
   }
 
-  function pumpSendToReply (request, reply, pathname) {
-    const stream = send(request.raw, pathname, sendOptions)
+  function pumpSendToReply (request, reply, pathname, rootPath) {
+    var options = Object.assign(sendOptions, {})
```

Comment:
```suggestion
    var options = Object.assign({}, sendOptions)
```

If we don't switch the parameter, the changes to options will reflect to sendOptions

Reply:
Oh good catch, originally I had this using the spread operator but that caused the CI tests to fail against node 6 so I changed it to this and put the parameters in the wrong order